packages/framework/src/types/discover.types.ts

```diff
@@ -67,6 +68,13 @@ export type DiscoverWorkflowOutput = {
   };
 };
 
+export type Workflow<T_Payload> = {
+  trigger: (
+    event: Prettify<Omit<EventTriggerParams<T_Payload>, 'workflowId' | 'bridgeUrl'>>
+  ) => Promise<EventTriggerResult>;
+  definition: DiscoverWorkflowOutput;
+};
```

Comment:
A wrapper around the defined workflow typings, enabling us to add usable methods like `trigger` to the workflow in a less cluttered namespace than `definition`.

I've chosen the `definition` naming on the thing we use for discovery as it conceptually makes sense as the noun for the workflow you defined in code. Additionally, `definition` will work well for other domains we add in the future. I would like to hear thoughts on this matter to verify it makes sense.